pkg/node/statestore.go

```diff
@@ -87,3 +96,30 @@ func overlayNonceExists(s storage.StateStorer) ([]byte, bool, error) {
 func setOverlayNonce(s storage.StateStorer, overlayNonce []byte) error {
 	return s.Put(OverlayNonce, overlayNonce)
 }
+
+func migrateStamperData(stateStore storage.StateStorer, stamperStore storage.Store) error {
+	var keys [][]byte
```

Comment:
Since the key below is cast to type `string`, I recommend using `var keys []string` instead and casting the key to type `string` on append.